nvi

```diff
@@ -149,13 +149,24 @@ install_node() {
 
   cd "$INSTALL_DIR/$PACKAGE"
   # Install executables in the bin dir.
-  cp -- "bin/node" "$BIN_DIR/node"
-  ln \
+  if ln \
```

Comment:
Below we have the command to run on the next line.
Wouldn't it make more sense to have that here also?

Reply:
The structure here is

```
if CONDITION
then
  STATEMENTS
else
  STATEMENTS
fi
```

so it's a question of whether we consider "CONDITION" special compared to "STATEMENTS". I personally do.

It's mostly incidental that the condition and statements look so similar. I think the suggested change would look really weird for code like the below.

```
if [[ $some_var ]]
then
  STATEMENTS
else
  STATEMENTS
fi
```

As I see it, it's a question of whether we want a special case formatting for this rather ungainly-looking if condition. I personally think an `if` with no following condition looks rather weird, too.